samples/CypressTest/cypress/script.js

```diff
@@ -3,24 +3,25 @@ const fse = require('fs-extra');
 const { merge } = require('mochawesome-merge');
 const generator = require('mochawesome-report-generator');
 
-const runTests= async ()=>{
+const runTests = async () => {
   await fse.remove('mochawesome-report');
   await fse.remove('cypress/report');
-  const {totalFailed} = await cypress.run();
+  const { totalFailed } = await cypress.run();
+  // To run tests with firefox comment the line above and uncomment the next one
+  // const {totalFailed} = await cypress.run({browser:'firefox'});
   const reporterOptions = {
     files: ["cypress/report/*.json"]
   };
   await generateReport(reporterOptions);
-  
-  if(totalFailed !== 0){
+
+  if (totalFailed !== 0) {
     process.exit(2);
   };
```

Comment:
The semicolon at the end of the if statement after the closing curly brace is unnecessary.